README.md

```diff
@@ -21,3 +21,11 @@ $ python weather_check.py 15 51.123456 -1.123456 --secret SomeSecretKeyHere!
 2017-01-06 09:58:17,943 Starting new HTTPS connection (1): api.darksky.net
 2017-01-06 09:58:18,350 You don't need to worry about salt tonight.
 ```
+
+## Prerequisites:
+
+This script requires Python and the argparse, logging & requests Python modules. For Ubuntu only the requests module is not included with the basic Python pacakges.  It can be installed by:
+
+```
+sudo apt install python python-requests
```

Comment:
> That is a matter of some debate on Ubuntu and Debian systems

For certain, but this script can be run on anything that supports python, including windows, OSX, linux and unix systems, where PIP is the only common python package manager (for python libraries anyway), so from a purely multi-OS perspective, it would be better to point users at that, no?

>  Apt just keeps things up to date for you relatively automatically. Does pip be default on Ubuntu?

In some python projects, having a specific dependency is a requirement.  If a package changed versions it could break something, its explicitly not wanted to auto upgrade.  As a result I don't know of any built-in automatic updates, though there are plenty of suggestions that would be cron-able...  I can specify in the `requirements.txt` file an explicit dependency such as `requests==2.11.1`, or support a range of versions etc.

In this particular instance it makes little difference since only the most basic parts of the requests library are being used and have remained stable for a long time, so I haven't specified a version (defaults to latest supported for the invoked version of python IIRC)